modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java

```diff
@@ -7,7 +7,7 @@
 /**
  * @author gusohal
  */
-public class OracleContainer extends JdbcDatabaseContainer {
+public class OracleContainer<SELF extends OracleContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
```

Comment:
It should be:
```suggestion
public class OracleContainer extends JdbcDatabaseContainer<OracleContainer> {
```

Reply:
So PostgreSQLContainer and MySQLContainer have also legacy class declaration, is it correct?

Reply:
yes, I think most of them

Reply:
Fixed